### PR TITLE
Add accessibility features

### PR DESCRIPTION
**Pull request by limavfabio**
This Pull Request adds features and improves existing content to enhance accessibility for the portfolio project.

Report:

Page titles: No issues found
Image text alternatives: Fixed issues (see more information as a comment on Files changed section)
Text headings: No issues found
Color contrast: No issues found
Resize: No issues found
Interaction: Fixed outline on keyboard focus for links and buttons; Add labels for forms inputs (see more information as a comment on Files changed section)
Moving content: No issues found
Multimedia: No issues found
The basic structure of the page: No issues found

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1,9 +1,29 @@
+label {
```
Comment (limavfabio) [inline style.css:1]: Make labels invisible for their purpose is to make the website more accessible for people using screen readers

File: style.css [hunk 2]
```diff
@@ -1,9 +1,29 @@
...
+  display: none;
+}
+
 a {
   color: inherit;
   text-decoration: inherit;
   fill: inherit;
 }
 
+li a:focus-visible {
```
Comment (limavfabio) [inline style.css:11]: Add a more contrasting outline when focusing links and buttons with keyboard

File: style.css [hunk 3]
```diff
@@ -558,6 +578,14 @@ form.send-message button {
   color: #fff;
 }
 
+form.send-message button:focus-visible {
```
Comment (limavfabio) [inline style.css:581]: Add a more contrasting outline when focusing form button with the keyboard for better visibility

**Review by BertrandConxy — CHANGES_REQUESTED** [review]
Hello @limavfabio and @Solobarine, 

Good job so far! 👏
I appreciate your efforts in making your portfolio accessible to a variety of assistive technologies but also to some people with disabilities. This helps to create something which is inclusive. However, there are some issues that you still need to work on to go to the next project but you are almost there! 🏃‍♂️ 

## Highlights 💪 
- NO linter errors
- Images have appropriate alt texts
- Descriptive PR and report in general
- correct GitHub flow is used.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 4]
```diff
@@ -1,14 +1,34 @@
...
 }
 
+button:focus-visible {
+  outline: auto;
+  outline-color: crimson;
+  outline-style: dashed;
+  outline-offset: 2px;
+  transition: none;
+}
+
```
[the remark this answers is not in the document]
Comment (limavfabio) [reply style.css:31]: Are these buttons contrast the only problems needing fixing? @BertrandConxy ?

**Pull request merged**